src/qt/sendcoinsdialog.cpp

```diff
@@ -319,9 +327,21 @@ void SendCoinsDialog::on_sendButton_clicked()
         formatted.append(recipientElement);
     }
 
-    QString questionString = tr("Are you sure you want to send?");
+    QString questionString;
+    if (!model->privateKeysDisabled()) {
+        questionString.append(tr("Are you sure you want to send?"));
+    } else {
+        questionString.append(tr("Do you want to draft this transaction?"));
+    }
+
     questionString.append("<br /><span style='font-size:10pt;'>");
-    questionString.append(tr("Please, review your transaction."));
+    QString pleaseReview;
+    if (!model->privateKeysDisabled()) {
+        pleaseReview = tr("Please, review your transaction.");
```

Comment:
9808f7867d9c557ba4bc253104f47fbd7f02b668

Avoid `pleaseReview` by calling `questionString.append` here and below? - like L332 above.